### PR TITLE
Shiv/translations page work

### PR DESCRIPTION
**Pull request by parikhshiv**
Adds early translations page, including api calls to firebase node server for retrieving full list of translations (will need to be paginated) and updating a translation.

### CODE REVIEW

File: src/translations/TranslationListItem.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import Fab from '@material-ui/core/Fab';
+import PlayIcon from '@material-ui/icons/PlayArrowOutlined';
+import ApiUtils from '../utils/ApiUtils';
+import './TranslationListItem.css';
+
+class TranslationListItem extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleTranslationChange_ = this.handleTranslationChange_.bind(this);
```
Comment (borismus) [inline src/translations/TranslationListItem.js]: Internal TS style prefers `() => {this.foo()}` to `foo.bind(this)`, but either is fine.

File: src/translations/TranslationListItem.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+  handleTransliterationChange_(e) {
+    this.setState({ transliteration: e.target.value });
+  }
+
+  saveTranslation_(e) {
+    const { english_word, sound_link, translation,
+      transliteration } = this.state;
+
+    fetch(`${ApiUtils.origin}${ApiUtils.path}addTranslations`, {
```
[the remark this answers is not in the document]
Comment (parikhshiv) [reply src/translations/TranslationListItem.js]: Fixed, converted over to async/await.

I replaced the ".catch"s with try/catch - not sure if there's a better solution to this though.

File: src/translations/TranslationListItem.js (hunk 1 above)
Comment (parikhshiv) [reply src/translations/TranslationListItem.js]: Updated.

File: src/translations/TranslationListItem.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (borismus) [reply src/translations/TranslationListItem.js]: The pattern is usually like

`
  try {
    let response = await fetch('http://no-such-url');
  } catch(err) {
    alert(err); // TypeError: failed to fetch
  }
`

[the remark this answers is not in the document]
Comment (parikhshiv) [reply src/translations/TranslationListItem.js]: ok great, that's what I went with (this is outdated).

File: src/translations/TranslationListItem.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        english_word,
+        sound_link,
+        translation,
+        transliteration,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      }
+    })
+    .then(resp => resp.json())
```
[the remark this answers is not in the document]
Comment (parikhshiv) [reply src/translations/TranslationListItem.js]: this is outdated, you'll need to view the changes from the latest commit (not sure why it doesn't update that automatically).

**Pull request merged**